crates/astria-sequencer-validation/src/proof.rs

```diff
@@ -41,13 +41,10 @@ impl MerkleTree {
         MerkleTree(tree)
     }
 
-    /// Returns the root hash of the merkle tree.
+    /// Returns the root hash of the merkle tree as a fixed sized array of 32 bytes.
     #[must_use]
     pub fn root(&self) -> [u8; 32] {
-        sha2::digest::Output::<Sha256>::from_slice(self.0.root().as_bytes())
-            .as_slice()
-            .try_into()
-            .expect("sha256 output is always 32 bytes")
+        (*self.0.root().as_bytes()).into()
```

Comment:
The above was unecessary. Clippy complained about this method not having a `# Panics`. No idea why this wasn't triggered earlier.